app/src/main/java/fr/free/nrw/commons/mwapi/UploadResult.java

```diff
@@ -36,13 +36,17 @@
 
     @Override
     public String toString() {
-        return "UploadResult{" +
-                "errorCode='" + errorCode + '\'' +
-                ", resultStatus='" + resultStatus + '\'' +
-                ", dateUploaded='" + dateUploaded.toString() + '\'' +
-                ", imageUrl='" + imageUrl + '\'' +
-                ", canonicalFilename='" + canonicalFilename + '\'' +
-                '}';
+        if (dateUploaded != null) {
```

Comment:
Do not add the check here. Add the check just for `dateuploaded`.

Reply:
ok

Reply:
I have added checking only for dateUpdate. check commit no 3